releasenotes/notes/fix-dijkstra-panic-3962ad36788cab00.yaml

```diff
@@ -0,0 +1,11 @@
+---
+fixes:
+  - |
+    Fixed an issue with the Dijkstra path functions:
+
+      * :func:`rustworkx.dijkstra_shortest_paths`
+      * :func:`rustworkx.dijkstra_shortest_path_lengths`
```

Comment:
Can you also expand this to:
* `rustworkx.bellman_ford_shortest_path_lengths`
*  `rustworkx.bellman_ford_shortest_paths`
* `rustworkx.astart_shortest_path_lengths`
*  `rustworkx.astar_shortest_paths`

Reply:
I didn't find an implementation of the `rustworkx.astart_shortest_path_lengths`, could you please  show me where is that function?

Reply:
I was mistaken, we only have a path function for A star